packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -56,18 +56,25 @@ enum _RefreshIndicatorMode {
 /// scrollable's contents and then complete the [Future] it returns. The refresh
 /// indicator disappears after the callback's [Future] has completed.
 ///
-/// If the [Scrollable] might not have enough content to overscroll, consider
-/// settings its `physics` property to [AlwaysScrollableScrollPhysics]:
+/// ## Troubleshooting
+///
+/// ### Refresh indicator does not show up
+///
+/// The [RefreshIndicator] requires the child to be scrollable. If the
+/// [Scrollable] child does not have enough content to overscroll, set its
+/// [Scrollable.physics] property to [AlwaysScrollableScrollPhysics]:
 ///
 /// ```dart
 /// ListView(
 ///   physics: const AlwaysScrollableScrollPhysics(),
 ///   children: ...
-//  )
+/// )
 /// ```
 ///
-/// Using [AlwaysScrollableScrollPhysics] will ensure that the scroll view is
-/// always scrollable and, therefore, can trigger the [RefreshIndicator].
+/// A [ScrollView] is not scrollable by default if the [ScrollView.controller]
```

Comment:
You might leave this comment out or reconsider it. The first point doesn't appear to be generally correct and the second point is maybe misleading - by default the refreshIndicator responds to its closest descendant per defaultScrollNotification and etc.

Reply:
I'll leave it this comment out. I think the above paragraph covers this now anyway.